cuda/btx_cudamatching_model.yaml

```diff
@@ -0,0 +1,58 @@
+:environment:
+  :entries:
+  - :name: ^hostname$
+    :type: string
+:stream_classes:
+- :default_clock_class: {}
+  :event_common_context_field_class:
+    :type: structure
+    :members:
+    - :name: ^vpid$
+      :field_class:
+        :type: integer_signed
+    - :name: ^vtid$
+      :field_class:
+        :type: integer_unsigned
+  :event_classes:
+  - :set_id: entries
+    :name: "_entry$"
+
+  - :set_id: exits
+    :register: false
+    :name: "_exit$"
+
+  - :set_id: exits_cudaError_present
+    :domain: exits
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+        :field_class:
+          :type: integer_signed
+          :cast_type: ^CUresult$
+
+  - :set_id: exits_cudaError_absent
+    :domain: exits - exits_cudaError_present 
+
+  # Memory traffic
+  - :set_id: traffic
+    :name: "cuMemcpy(2D|3D)?(Unaligned)?(DtoH|DtoD|HtoD)(Aync)?_v2(_ptds|_ptsz)?"
```

Comment:
Doesn'y `^cuMemcpy` work? or you are afraid of some of the memcopy should be ignored?